data/filters/templates/woolworths-cleanup.yaml

```
@@ -0,0 +1,38 @@
+title: "Woolworths (AU): Cleanup search results"
+contributors:
+  - JohnyP36
+  - sammcj
+  - xvello
+params:
+  - name: market-partners
+    description: Hide reseller items from market partners in search results
+    type: checkbox
+    default: true
+  - name: non-product-titles
+    description: Hide non-product titles in search results
+    type: checkbox
+    default: true
+tags:
+  - woolworths
+  - australia
+template: |
+  {{#if market-partners}}
+  www.woolworths.com.au###search-content ##[class*="market-product-message-wrapper"]
```

This does not look like a valid uBO syntax due to the `##` in the middle, what is your intent here? 

```suggestion
  www.woolworths.com.au###search-content shared-product-vendor-message:upward(div.product-grid-v2--tile)
```